packages/php-wasm/universal/src/lib/php-request-handler.ts

```diff
@@ -194,6 +222,10 @@ export class PHPRequestHandler {
 			this.#PATHNAME,
 		].join('');
 		this.rewriteRules = rewriteRules;
+
+		if (config.getFileNotFoundAction) {
```

Comment:
Nitpick: the default handler could be a default option vslue - we wouldn’t need this if

Reply:
Ah, good point. I see how we are destructuring the config arg above.

Reply:
> Nitpick: the default handler could be a default option vslue - we wouldn’t need this if

This was a great nit to fix. Done!